etc/metricshipper.yaml

```diff
@@ -75,3 +75,7 @@
 # Set the glog logging verbosity
 # 
 #verbosity: 0
+
+# Enable or disable mtrace
+#
+#mtraceenabled: false
```

Comment:
William asked for this as a performance enhancement. When mtrace is enabled, the code has to search each batch for the mtrace flag, which requires an iteration through the batch that is otherwise unneccessary. This flag should be false in production (for performance), and set to enable mtrace functionality only when mtrace is added at the collector.